### PR TITLE
Add new people

### PR DESCRIPTION
**Pull request by markowanga**

### CODE REVIEW

**Comment by kajdanowicz** [comment]
- Rajmund should have role: Alumni PhD
- Rajmund should have additional eduction: 2023 - PhD in information and communication technology, Wroclaw University of Science and Technology
- Arkadiusz should also have 2nd role: Alumni PhD
- Arkariusz should habe additional education: 2023 - PhD in information and communication technology, Wroclaw University of Science and Technology

File: content/authors/janz/_index.md [hunk 1]
```diff
@@ -51,8 +51,7 @@ email: "arkadiusz.janz@pwr.edu.pl"
 # Organizational groups that you belong to (for People widget)
 #   Set this to `[]` or comment out if you are not using People widget.
 user_groups:
-  - PhD Students
-  - Project's staff
+  - Faculty
 ---
```
Comment (kajdanowicz) [inline content/authors/janz/_index.md:61]: Alumni PhD

**Comment by markowanga** [comment]
Ok, I'm adding right now

**Comment by markowanga** [comment]
I have also downgraded Janz priority -- he was first on page 😄

**Comment by kajdanowicz** [comment]
Order of the faculty staf should ne
Kajdanowicz
Bródka
Zięba
Kazienko
Klonowski
Kwasnicka
Markowska-Kaczmar
Chodak
Michalski
Piasecki
Saganowski
Janz
Kocon
Sas
Syga
Szymański
Tabakow
Walkowiak
Dolata
Jakubik
Zawisza

**Comment by kajdanowicz** [comment]
Kajdanowicz shoul have removed "head of faculty’s doctoral studies"

**Comment by kajdanowicz** [comment]
Pleas add anything about Grzegorz Chodak based on details from https://p.wz.pwr.edu.pl/~chodak.grzegorz/

**Comment by kajdanowicz** [comment]
Arkadiusz Janz shold have Assistant Professor in Faculty role and Alumni PhD in Alumni Phd role. If it is not possible to make such split, please make him "Assistant Professor, Alumni PhD" in both places

**Comment by markowanga** [comment]
> Arkadiusz Janz shold have Assistant Professor in Faculty role and Alumni PhD in Alumni Phd role. If it is not possible to make such split, please make him "Assistant Professor, Alumni PhD" in both places

I can set only one role description (it's global)

**Comment by kajdanowicz** [comment]
Phd students should be in the following order (alphabetical order on last name):
Baran
Bielaniewicz
Binkowski
Czuba
Dazhunts
Dykiert
Dziubałtowska
Ferdinan
Fojcik
Gramacki
Jabłońska
Jakubowska
Janiak
Kanclerz
Karol
Kawa
Komoszyńska
Koptyra
Kościukiewicz
Kozłowski
Kunc
Marciniak
Migacz
Miłkowski
Nurek
Perz
Plata
Rajda
Sawczyn
Serwata
Stępień
Szołomicka
Tagowski
Walentynowicz
Wielopolski
Wojtasik
Woźniak
Wójcik
Zieliński

**Comment by kajdanowicz** [comment]
Additional work on phd students:
- please remove totally from phd students Artur Zawisza (keep in faculty role), Ilya Hradovich (remove),  Artur Budniak (remove), Maksymilian Gut (remove)
- move Piotr Bielak do alumni Phd role

**Comment by markowanga** [comment]
Szybko poszło

**Comment by markowanga** [comment]
@kajdanowicz look for changes -- imo ready for merge

**Review by kajdanowicz — APPROVED** [review]

**Pull request merged**